packaging/packager_rpm.sh

```diff
@@ -12,30 +12,33 @@ dnf -y install python3 python3-coverage
 dnf -y install python3-pillow
 # pip dependencies (for dependencies not available as RPM)
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+
 pip3 install --upgrade pip
+pip3 install setuptools setuptools-rust
+pip3 install pillow==10.4.0
+
 # contour, template, feature, cascade, text matching
 dnf -y install python3-numpy python3-opencv
+pip3 install numpy opencv-python
+
 # text matching
 dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
-pip3 install pytesseract==0.3.4 tesserocr==2.5.1
+pip3 install pytesseract==0.3.10 tesserocr==2.7.0
+
 # deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
+pip3 install torch==2.2.0 torchvision==0.17.0
+
 # screen controlling
-if (( distro_version <= 32 )); then
-    pip3 install autopy==4.0.0
-else
-    export DISABLE_AUTOPY=1
-fi
+export DISABLE_AUTOPY=1
 # TODO: vncdotool doesn't control its Twisted which doesn't control its "incremental" dependency
 pip3 install incremental==22.10.0
 pip3 install vncdotool==0.12.0
 dnf -y install xdotool xwd ImageMagick
 # NOTE: PyAutoGUI's scrot dependencies are broken on Fedora 33- so we don't support these
-dnf -y install python3-tkinter scrot
-# TODO: install PyScreeze separately to replace the one PyAutoGUI will install which is incompatible with the current Pillow version
-pip3 install pyscreeze==0.1.28
-pip3 install pyautogui==0.9.53
+dnf -y install python3-tkinter
+dnf -y install gnome-screenshot
```

Comment:
So gnome-screenshot installs pyscreeze or needed stuff by pyautogui?

Reply:
If we have the gnome-screenshot pyscreeze is not needed (pyautogui will choose between the two). only installing pyscreeze was not working